src/components/Tables/Table.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import React, { useState, useMemo } from "react";
+import DataTable, { TableColumn } from "react-data-table-component";
+
+interface Props {
+    data: any[];
```

Comment:
Worth adding an interface for data or no? To use instead of any[].

Reply:
We have replaced it with a more specific type (while generic enough for all JSON from Schema)

data: { [key: string]: string | number | null }[];